pandoc.cabal

```diff
@@ -282,6 +283,8 @@ Flag old-locale
 Library
   Build-Depends: base >= 4.7 && < 5,
                  syb >= 0.1 && < 0.8,
+                 split == 0.2.3.2,
+                 HUnit == 1.6.0.0,
```

Comment:
Why a dependency on HUnit?
We have this dependency in the test suite; it shouldn't be needed in normal library code.